401-react-clickable-grid/src/js/GridBox.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * n × n 单元格组件
+ */
+
+import GridBoxColumn from './GridBoxColumn';
+
+var GridBox = React.createClass({
+    getInitialState: function () {
+        return {rowNum: this.props.rowNum};
+    },
+
+    componentWillReceiveProps: function (nextProps) {
```

Comment:
e...I use it just to practise how to avoid unnecessary state changes. maybe needless